### ISSUE TITLE
Scraping error: login.2fa.trusted-dev.invalid

### ISSUE THREAD

**Issue by jwilk** [jwilk/mbank-cli#81]
If you try to register device under a name that's already taken, you get:

```
Scraping error: login.2fa.trusted-dev.invalid at ./mbank-cli line 1187.
        main::_do_2fa("CLI") called at ./mbank-cli line 1427
        main::do_login("register_device", "CLI") called at ./mbank-cli line 1474
        main::do_register_device("args", ARRAY(0x1f0e658)) called at ./mbank-cli line 3000
```

The response body is:
```json
{"isValid":false}
```

So this should be straight-forward to fix… Except that the mBank UI doesn't offer an option to retract from device registration, or even to log out at this point. The only option it offers is to enter a different name.

I guess we could ask the user for a new name interactively? Feels icky, but it should work.

**Comment by chilek** [comment]
> So this should be straight-forward to fix… Except that the mBank UI doesn't offer an option to retract from device registration, or even to log out at this point. The only option it offers is to enter a different name.

They promise to improve web user interface in the field of registered device management, but don't specify when.

**Comment by jwilk** [comment]
Fixed in [mbank-cli 2.2.1](https://github.com/jwilk/mbank-cli/releases/tag/2.2.1).
